app/controllers/bookings_controller.rb

```diff
@@ -61,14 +85,32 @@ def destroy
     end
   end
 
+  def get_city_wise_cleaner
+    @cleaner_temp = CitiesCleaner.where(city_id:City.find_by(name:params[:city]))
+    @cleaner = []
+    @cleaner_temp.each do |cl|
```

Comment:
@mayur-kambariya Can we not use `.inject` instead of this, it would optimize the code a bit.